lms/static/scripts/frontend_apps/components/BasicLTILaunchApp.js

```diff
@@ -46,14 +46,14 @@ export default function BasicLTILaunchApp() {
       // API callback to use to fetch the URL to show in the iframe. This is
       // needed if resolving the content URL involves potentially slow calls
       // to third party APIs (eg. the LMS's file storage).
-      viaUrl: viaUrlApi,
+      viaUrl: viaAPICallInfo,
       // Sync API callback and data to asynchronously load the section groups
       // to relay to the sidebar via RPC.
-      sync: apiSync,
+      sync: groupAPICallInfo,
```

Comment:
The settings that the backend provides and the alias we're giving it seem to be using different terminology ("sync" vs "groups"). On the backend the route is called `canvas_api.sync` and the configuration for the frontend is filled out in a method called `_sync_api`. How about calling this `syncAPICallInfo`?

Reply:
I have an overall confusion as to what we mean when we say "sync API". I can make this change, I'm just not clear on what it's supposed to mean overall. Is the idea that there could be other services (besides fetching groups) that the API provides in the future? What are we "synching?" Like, what _is_ the sync API?